### DISCUSSION TITLE
Support for image-rendering: pixelated

### DISCUSSION THREAD

**Discussion by brunowowk** [vercel/satori#619] [Ideas]
# Feature Request

## Description

When using upscaled png images, specifically ones including pixel art, I want to be able to use the css style `image-rendering: pixelated` to make the image crisp (make it use nearest neighbour scaling)

## Additional Context

Currently that style has no effect, and upscaled pixel art images get blurry. The workaround is upscaling them with something like https://github.com/lovell/sharp before using with Satori.

**Comment by shuding** [comment]
I don't think there's an equivalent API inside SVG's `<image>` element (https://developer.mozilla.org/en-US/docs/Web/SVG/Element/image), so the end result of Satori can't have pixelated images embedded. Thanks for the idea though, I'll move this to discussions until we have a clear path on how this can be supported.
